src/main/features/dashboard/views/defendant.njk

```diff
@@ -14,28 +14,38 @@
     <div class="column-two-thirds">
       <h2 class="heading-medium">{{ t('Claim status') }}</h2>
 
-      {{ responseStatusForDefendant(claim, DashboardUrlHelper.getStatusUrl(claim), isAfter4pm) }}
-
       {% if featureToggles.offer and claim.status === ClaimStatus.OFFER_ACCEPTED %}
-        <p>
-          <a href="{{ OfferPaths.countersignAgreementPage.evaluateUri({ externalId: claim.externalId }) }}">
-            {{ t('Sign the agreement') }}
-          </a>
-        </p>
-      {% elif featureToggles.offer and not claim.settlementReachedAt  %}
-        <h2 class="heading-medium">{{ t('More options') }}</h2>
-        <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
-        <p>{{ t('You can still <a href="%s">make an offer</a> to settle the claim out of court. You will avoid a hearing and a County Court Judgment against you.',
-                { postProcess: 'sprintf',
-                  sprintf: [ OfferPaths.offerPage.evaluateUri({ externalId: claim.externalId }) ]
-                }) | safe
-            }}
-        </p>
+        {% for defendantClaimState in claim.stateHistory %}
+          <div class="status-block">
+            <p>{{ (defendantClaimState.date | date) if defendantClaimState.date}}</p>
+            <p>
+              {{ responseStatusForDefendantBasedOnState(defendantClaimState, claim, DashboardUrlHelper.getStatusUrl(claim), isAfter4pm) }}
+            </p>
+          </div>
+        {% endfor %}
+
+        {% elif featureToggles.offer and claim.status === ClaimStatus.OFFER_SETTLEMENT_REACHED %}
+        <p>{{ claim.respondedAt | date }}</p>
+        <h3 class="heading-small">{{ t('Settle Claim') }} </h3>
+        {{ responseStatusForDefendant(claim, DashboardUrlHelper.getStatusUrl(claim), isAfter4pm) }}
+        <a href="{{ claimSettlementUri }}">{{ t('Download the agreement') }}</a>
+
+        {% elif featureToggles.offer and not claim.settlementReachedAt %}
```

Comment:
Is `and not claim.settlementReachedAt` required here?

Reply:
...that was existing before. I didn't want to make any changes there without completely understanding. If you want i can spend more time on it and make necessary changes.